ios/Runner/Info.plist

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+    <key>BGTaskSchedulerPermittedIdentifiers</key>
```

Comment:
This only fixes things on iOS 13 plus apparently when running on my iPhone 5s it fails. I think we should move this to the AppDelegate though I haven't found a  way yet.